code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -704,8 +702,8 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	desc = "Creates a small shell fitted for soul vessels. Adding an active soul vessel to it results in a small construct with tools and an inbuilt proselytizer."
 	invocations = list("Pnyy sbegu...", "...gur jbexref-bs Nezbere.")
 	channel_time = 50
-	required_components = list("belligerent_eye" = 2, "hierophant_ansible" = 1)
-	consumed_components = list("belligerent_eye" = 1, "hierophant_ansible" = 1)
+	required_components = list("belligerent_eye" = 2, "hierophant_ansible" = 1, "replicant_alloy" = 3)
+	consumed_components = list("belligerent_eye" = 1, "hierophant_ansible" = 1, "replicant_alloy" = 2)
```

Comment:
If the issue is combat, nerfing costs won't help, it'll just make them less common and still dangerous. Have you seen any cogscarabs do well in combat since the nerf?